Source/gs/GSHandler.h

```diff
@@ -234,6 +234,15 @@ class CGSHandler
 		REGISTER_MAX = 0x80
 	};
 
+	enum GS_HANDLERS
```

Comment:
Enum is good, but it should reside in the Qt UI module. CGSHandler doesn't need to know about its implementations.

Reply:
sure, but i presume you'd want android to get vulkan as well, so if it went into lets say `SettingsDialog`, you'd need another enum on android side

Reply:
True. When we get there, it would probably make sense to have it in `ui_shared`, but let's keep it in `ui_qt` for now.